### PR TITLE
Improve and contain small screens responsive layout

### PR DESCRIPTION
**Pull request by MattiSG**
On small screens, the widget was still impacting the global layout.

### CODE REVIEW

**Comment by laem** [comment]
Nice !

File: css/main.css [hunk 1]
```diff
@@ -86,14 +86,12 @@ details {
 	display: none;
 }
 
-@media(max-width: 500px) {
-	body {
-		padding: 0;
-		width: 15em;
+@media(max-width: 600px) {
+	#input, #result {
 		text-align: left;
```
Comment (laem) [inline css/main.css]: I think the original `text-align: center` is more beautiful even on small screens, especially given the nulled select and input margins.

Comment (MattiSG) [inline css/main.css]: No strong opinion. This wasn't a change though, is this blocking for merging?

Comment (laem) [inline css/main.css]: It was not being applied as the following non media rule was more specific : 
```css
#input, #error, #result, summary {
	text-align: center;
}
```

Comment (MattiSG) [inline css/main.css]: Not applying to `#error` and `summary` was voluntary  ;)

**Pull request closed without merging**